imageproxy.go

```diff
@@ -131,7 +131,8 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	copyHeader(w, resp, "Last-Modified")
 	copyHeader(w, resp, "Expires")
 	copyHeader(w, resp, "Etag")
-
+	copyHeader(w, resp, "Link")
+	
```

Comment:
remove trailing space on this line.  (or just run gofmt which will do this for you)